src/shared/ui/survey/RadioActivityItem/RadioItem.tsx

```diff
@@ -0,0 +1,73 @@
+import { FC } from 'react';
+
+import { styled } from '@tamagui/core';
+
+import { colors } from '@shared/lib';
+import {
+  XStack,
+  RadioGroup,
+  Text,
+  Box,
+  Image,
+  QuestionTooltipIcon,
+  Tooltip,
+} from '@shared/ui';
+
+import RadioOption from './types';
+
+type RadioLabelProps = {
+  option: RadioOption;
+};
+
+const RadioTooltipContainer = styled(Box, {
+  marginRight: 10,
+  width: 36,
+});
+
+const RadioTextContainer = styled(Box, {
+  marginLeft: 10,
+  flexGrow: 1,
+});
+
+const RadioItem: FC<RadioLabelProps> = ({
+  option: { isVisible, name, value, image, description },
+}) => {
+  if (!isVisible) {
+    return null;
+  }
+
+  return (
+    <XStack width="100%" paddingVertical={5} alignItems="center">
```

Comment:
Paddings should be set on RadioActivityItem level

Reply:
no need to set width 100% here

Reply:
done